src/Command/DrupalUpdate/DrupalUpdateCommand.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+
+namespace Acquia\Cli\Command\DrupalUpdate;
+
+use Acquia\Cli\Command\CommandBase;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class DrupalUpdateCommand extends  CommandBase
+{
+  protected static $defaultName = 'd7-update';
```

Comment:
This isn't following the existing patterns for commands names. Seems like it should be something like `app:update-packages`